modules/Makefile

```diff
@@ -31,7 +31,7 @@ tests:
 	@docker exec -t cb-test bin/bash -c "cd modules && bats -T ./test"
 
 # Run a single test.
-# i.e make TEST_NAME="Analytics" single-test
+# i.e make TEST_NAME="Analytics.java" single-test
```

Comment:
Why the new `.java` suffix in these btw?

Reply:
I'd actually forgotten to put that in in the initial PR, we had that in the Go one for example, so trying to keep it consistent.